mods/ts/rules/civilian-structures.yaml

```diff
@@ -722,6 +722,14 @@ CAARAY:
 		HP: 400
 	RenderSprites:
 		Palette: player
+	WithIdleOverlay@SATELITTE:
+		Sequence: idle-satelitte
+	WithIdleOverlay@RADAR:
+		Sequence: idle-radar
+	WithIdleOverlay@SATELITTE:
```

Comment:
You used `@SATELITTE` twice, which is why one of the animations doesn't show. Change this to `@SCANNER`.